module-presentation/src/main/kotlin/site/yourevents/health/api/HealthApi.kt

```diff
@@ -0,0 +1,16 @@
+package site.yourevents.health.api
+
+import DisableSwaggerSecurity
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.tags.Tag
+import org.springframework.web.bind.annotation.GetMapping
+import site.yourevents.response.ApiResponse
+
+@Tag(name = "상태 확인")
+interface HealthApi {
+
+    @DisableSwaggerSecurity
```

Comment:
마찬가지로, 컨벤션 수정 부탁드립니다!
```suggestion
interface HealthApi {
    @DisableSwaggerSecurity
```